Makefile

```diff
@@ -442,7 +442,7 @@ qemu-version.h: FORCE
 			pkgvers="$(PKGVERSION)"; \
 		else \
 			if test -d .git; then \
-				pkgvers=$$(git describe --match 'v*' 2>/dev/null | tr -d '\n');\
+				pkgvers=$$(git rev-parse --short HEAD 2>/dev/null | tr -d '\n');\
 				if ! git diff-index --quiet HEAD &>/dev/null; then \
```

Comment:
This command never runs cleanly for me, I get the help message because syntax is wrong.
Can anyone else confirm?

I've also checked with [upstream](https://git.qemu.org/?p=qemu.git;a=blob;f=Makefile;h=7ed9cc4a211aef25e30bb4b2354308d2c18a46c3;hb=HEAD#l449), same command there.

Reply:
I can not reproduce this anymore. Must have been a bug stemming from a software upgrade or something.